server/db/methods/StretchAnswer.js

```diff
@@ -1,37 +1,90 @@
 const { Op } = require('sequelize')
-const { User, CohortStretch, CohortUser, StretchAnswer } = require('../models')
+const {
+  User,
+  CohortStretch,
+  CohortUser,
+  StretchAnswer,
+  Cohort
+} = require('../models')
 
-const getAnswersOfStudentsOfSingleAdmin = function(adminId) {
+// These are the main parameters that are used in Sequelize models' find methods.
+const defaults = {
+  include: [
+    {
+      model: CohortStretch,
+      attributes: ['cohortId', 'stretchId'],
+      include: [
+        {
+          model: Cohort,
+          attributes: ['name']
+        }
+      ]
+    }
+  ]
+}
+
+// This method formats a single instance of StretchAnswer.
+StretchAnswer.prototype.format = function() {
+  const { cohortstretch, ...stretchAnswerFields } = this.dataValues
+  const { cohort, ...cohortsStretchFields } = cohortstretch.get()
+  return {
+    ...stretchAnswerFields,
+    ...cohortsStretchFields,
+    cohortName: cohort.name
+  }
+}
+
+StretchAnswer.prototype.addAssociations = function(format = true) {
+  return StretchAnswer.findByPk(this.id, defaults).then(sa =>
+    format ? sa.format() : sa
+  )
+}
+
+StretchAnswer.getAnswersOfStudent = function(studentId) {
+  return this.findAll({
+    where: { userId: studentId },
+    ...defaults
+  }).then(stretchAnswers => {
+    return stretchAnswers.map(sa => sa.format())
+  })
+}
+
+StretchAnswer.getAnswersOfStudentsOfSingleAdmin = function(adminId) {
```

Comment:
Hi, are the ``StretchAnswer`` objects in this function and ``Stretch.getAnswersOfStudent`` both being stored in the ``stretchAnswers`` array in the Redux store? It looks like the objects slightly vary in formatting.